cmd/cluster-network-operator/k8sutil.go

```diff
@@ -40,7 +40,7 @@ const (
 	ClusterRunMode RunModeType = "cluster"
 )
 
-var log = logf.Log.WithName("k8sutil")
+var logger = logf.Log.WithName("k8sutil")
```

Comment:
Delete this logger; we don't need this fine-grained logging.

Reply:
ack